### ISSUE TITLE
One final request on the port

### ISSUE THREAD

**Issue by Dededeez** [TurtleHouse/PizzaTowerSwitch-FMOD#50]
I know you said that you're finished with it, but with that being the case, could you possibly do one last update that removes connecting to the internet to check for updates, seeing as the port won't be receiving anymore updates? This is entirely unnecessary, but it would be slightly convenient for anyone who still would like to play the unofficial port. Once again, I can't thank you enough for all your work on the unofficial port, and good luck to anything else you do in the future.

**Comment by D-Official** [comment]
Nah, too lazy. But I should probably note that I plan to keep the update server for as long as I can, and it being up inadvertently makes launching the port faster.

**Comment by Dededeez** [comment]
Huh, interesting, didn't know that. Actually, I have another question. In the official port, there are actually slight slowdowns in similar places as your port, such as launching from the Trash Pan as The Noise, or going near the level gate to Pizzascare, and even occasional crashes on Deep Dish-9 and Oh Shit. Is there any sort of technical explanation as to why they may share similar slowdowns and crashes in those places? Is that related to the Switch's hardware or something?

**Comment by D-Official** [comment]
I can speculate that the slowdown from launching from the Trash Pan in Oh Shit is because the sprites related to that aren't in the correct texture group, resulting in them getting loaded on-demand instead of at the start (but again, I haven't checked). The stutter near the Pizzascare level entrance has been there since SR 1, I also think it's some texture that's not in the correct group. I'm surprised to hear it made it into the official port!

As I understand it, you don't really notice issues of this type happening on PC because it's just faster.

As for crashes in levels - the only similar crashes I can think of that I encountered were when trying to load the textures of DMAS and RRF when trying to enter them, since those stages store their massive background animations in an incredibly inefficient way. But never anything with those levels.
